build.sh

```diff
@@ -781,10 +781,10 @@ fi
 if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
     if [ $__CrossBuild == 1 ]; then
         __ClangMajorVersion=3
-        __ClangMinorVersion=6
+        __ClangMinorVersion=9
     else
         __ClangMajorVersion=3
-        __ClangMinorVersion=5
+        __ClangMinorVersion=9
```

Comment:
This is just a testing PR, not a final PR for the switch. You are right of course that after we make the change, we would remove the if / else.